javaparser-core/src/main/javacc_support/com/github/javaparser/GeneratedJavaParserSupport.java

```diff
@@ -148,4 +144,63 @@ static JavaToken nodeListBegin(NodeList<?> l) {
         }
         return l.get(0).getTokenRange().get().getBegin();
     }
+
+    /**
+     * This is the code from ParseException.initialise, modified to be more horizontal.
+     */
+    static String makeMessageForParseException(ParseException exception) {
+        final StringBuilder sb = new StringBuilder("Parse error. Found ");
+        final StringBuilder expected = new StringBuilder();
+
+        int maxExpectedTokenSequenceLength = 0;
+        TreeSet<String> sortedOptions = new TreeSet<>();
+        for (int i = 0; i < exception.expectedTokenSequences.length; i++) {
+            if (maxExpectedTokenSequenceLength < exception.expectedTokenSequences[i].length) {
+                maxExpectedTokenSequenceLength = exception.expectedTokenSequences[i].length;
+            }
+            for (int j = 0; j < exception.expectedTokenSequences[i].length; j++) {
```

Comment:
"I could, but should I? This is copied from the JavaCC source. Maybe it should be an issue on their project."